plutus-chain-index-core/plutus-chain-index-core.cabal

```diff
@@ -1,147 +1,162 @@
 cabal-version: 2.2
-name: plutus-chain-index-core
-version: 0.1.0.0
-license: Apache-2.0
+name:          plutus-chain-index-core
+version:       0.1.0.0
+license:       Apache-2.0
 license-files:
   LICENSE
   NOTICE
-maintainer: sjoerd.visscher@iohk.io
-author: Sjoerd Visscher
-homepage: https://github.com/iohk/plutus#readme
-bug-reports: https://github.com/iohk/plutus/issues
+
+maintainer:    sjoerd.visscher@iohk.io
+author:        Sjoerd Visscher
+homepage:      https://github.com/iohk/plutus#readme
+bug-reports:   https://github.com/iohk/plutus/issues
```

Comment:
Can you also change it to the correct link?
`https://github.com/input-output-hk/plutus-apps[...]`

Reply:
Not resolved